core/src/main/java/com/google/cloud/sql/core/InternalConnectorRegistry.java

```diff
@@ -261,4 +267,55 @@ private Connector createConnector(ConnectionConfig config) {
         refreshTimeoutMs,
         serverProxyPort);
   }
+
+  private ConnectionConfig resolveConfig(ConnectionConfig config) {
+    if (config.getNamedConnection() != null) {
+      ConnectorKey namedConfig = resolveKey(config);
```

Comment:
rename to `connectorKey` or something similar

Reply:
Fixed.